assets/tldr/teaser.ipynb

```diff
@@ -48,9 +48,7 @@
     "with_torch_launch = \"WORLD_SIZE\" in os.environ\n",
     "\n",
     "if in_colab:\n",
-    "    VERSION = \"nightly\"\n",
-    "    !curl https://raw.githubusercontent.com/pytorch/xla/master/contrib/scripts/env-setup.py -o pytorch-xla-env-setup.py\n",
-    "    !python pytorch-xla-env-setup.py --version $VERSION"
+    "    !pip install cloud-tpu-client==0.10 https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.8.1-cp37-cp37m-linux_x86_64.whl"
```

Comment:
Can't we install just latest stable instead of pinning the version to 1.8.1 ?

Reply:
Second option is OK for me, thanks 👍

Reply:
Done! Is it alright now?

Reply:
@Priyansi have you tested it on Colab ?

Looks like it does not work : https://colab.research.google.com/github/pytorch/ignite/blob/3dce762c089f785eb103f6e2cd5e2961673950f6/assets/tldr/teaser.ipynb#scrollTo=Y6x4WuLQ3j2a

Reply:
Oh my bad. I accidentally put a `$` before VERSION. Fixing it ASAP and running the entire notebook again.

Reply:
Btw, I think there could be a more simple/shorter way to fetch tag_name from the curl output...

Reply:
If we are looking for a shorter regex, this would be the best option, according to me - `VERSION = !curl -s https://api.github.com/repos/pytorch/xla/releases/latest | grep -Po '"tag_name": "v\K.*?(?=")'`

Reply:
Yes, I was thinking that too but it requires extra overhead that `grep` could easily solve. I made the above change. It should work now.